python-project-template/{% if include_benchmarks %}benchmarks{% endif %}/asv.conf.json.jinja

```diff
@@ -0,0 +1,77 @@
+{
+    // The version of the config file format.  Do not change, unless
+    // you know what you are doing.
+    "version": 1,
+    // The name of the project being benchmarked
+    "project": "{{project_name}}",
+    // The project's homepage
+    "project_url": "https://github.com/lincc-frameworks/{{project_name}}",
+    // The URL or local path of the source code repository for the
+    // project being benchmarked
+    "repo": "..",
+    // List of branches to benchmark. If not provided, defaults to "master"
+    // (for git) or "tip" (for mercurial).
+    "branches": [
+        "HEAD"
+    ],
+    "build_command": [
+        "python -m build --wheel -o {build_cache_dir} {build_dir}"
+    ],
+    // The DVCS being used.  If not set, it will be automatically
+    // determined from "repo" by looking at the protocol in the URL
+    // (if remote), or by looking for special directories, such as
+    // ".git" (if local).
+    "dvcs": "git",
+    // The tool to use to create environments.  May be "conda",
+    // "virtualenv" or other value depending on the plugins in use.
+    // If missing or the empty string, the tool will be automatically
+    // determined by looking for tools on the PATH environment
+    // variable.
+    "environment_type": "virtualenv",
+    // the base URL to show a commit for the project.
+    "show_commit_url": "https://github.com/lincc-frameworks/{{project_name}}/commit/",
```

Comment:
This assumes projects live under lincc-frameworks. Should we create a copier question to ask for the name of the organization?

Reply:
I understand! I will include adequate setup instructions in the RTD documentation and mention the need to change "show_commit_url" and "project_url".